check_rpmspec_collection.sh

```diff
@@ -41,7 +42,7 @@ run() {
 }
 
 run "--without ansible" "without_ansible" 1
-run "--without ansible --with collection_artifact" "without_ansible-with_collection_artifact" 1
+run "--without ansible --with collection_artifact" "without_ansible-with_collection_artifact" 0
 run "--without ansible --without collection_artifact" "without_ansible-without_collection_artifact" 1
 run "--without collection_artifact" "without_collection_artifact" 1
 run "--with ansbile --without collection_artifact" "with_ansible-without_collection_artifact" 1
```

Comment:
There are a couple of places where you use `ansbile` instead of `ansible` that should be fixed

Reply:
Thank you, @richm. There were plenty of them... :(